Update standard library DUI release note (#621)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta2-release-note.md

### Standard Library Updates

> The new PostgreSQL database package is introduced with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.

Reviewer: ```suggestion
The new `postgresql` database package is introduced with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.
```

#### New Features

##### Log Package

> Added Observability span context values to log messages when Observability is enabled.

Reviewer: ```suggestion
Added Observability span context values to log messages when observability is enabled.
```

##### I/O Package

> Introduced `io:fprint` and `io:fprintln` APIs.

Reviewer: ```suggestion
Introduced the `io:fprint` and `io:fprintln` APIs.
```

```ballerina
io:fprint(io:stderr, "Unexpected error occurred");
io:fprintln(io:stderr, "Unexpected error occurred");
io:fprint(io:stdout, "Passed without an error");
io:fprintln(io:stdout, "Passed without an error");
```

#### Improvements

##### Database Packages

> Returned type of the stream is now inferred as a second parameter to the query remote method.

Reviewer: ```suggestion
Return type of the stream is now inferred as a second parameter to the query remote method.
```

> For more info see [this issue](https://github.com/ballerina-platform/ballerina-standard-library/issues/1445)

Reviewer: ```suggestion
For more info, see [this issue](Processing streams in database operations are bit confusing).
```
normally, we do not link to issues like this in the Release Note.

Author: @lahirusamith Can you check this?

Reviewer 2: @kalaiyarasiganeshalingam, as we discussed, shall we add a description here rather than an issue link. ?

### Standard Library Updates

The new `postgresql` database package is introduced with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.

#### New Features

##### Log Package
Added Observability span context values to log messages when observability is enabled.

##### I/O Package
Introduced the `io:fprint` and `io:fprintln` APIs.
```ballerina
io:fprint(io:stderr, "Unexpected error occurred");
io:fprintln(io:stderr, "Unexpected error occurred");
io:fprint(io:stdout, "Passed without an error");
io:fprintln(io:stdout, "Passed without an error");
```

#### Improvements

##### Database Packages

> Return type of the stream is now inferred as a second parameter to the query remote method.

Author: @praneesha @lahirusamith I have changed the database package note. Please review

Reviewer 2: LGTM!
